election/github_emails_2.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+"""get_emails_2.py Extracts github IDs from a CSV and matches them
```

Comment:
any reason not to replace the other script?

Reply:
You can run this first on the output of grafana, and then run the other script on the missing list produced by this script.
Reason being that the first script is more likely to give clean results, but also to have many miss.
I should have documented this better? Or maybe aggregated the two steps? I was a bit lazy :P